java/registry/src/main/java/dev/sunbirdrc/registry/controller/RegistryEntityController.java

```diff
@@ -245,6 +248,11 @@ public ResponseEntity<Object> postEntity(
         newRootNode.set(entityName, rootNode);
 
         try {
+            if(!entityName.equals("Schema")) {
+                entityParenter.ensureKnownParenters();
+                entityParenter.loadDefinitionIndex();
+                entityParenter.ensureIndexExists();
```

Comment:
this is triggered for every add entity and this will loop through all the schemas and start creating the index for everything,
which is not the right way and it will have a high impact

Reply:
move this to schemaservice

Reply:
I moved this to SchemaService in addSchema but ndex and uniqueIndex is not created.